src/main/java/com/google/step/YOUR_PROJECT_NAME_HERE/data/Card.java

```diff
@@ -6,13 +6,6 @@
   private String link;
   private String description;
 
```

Comment:
since we specified a constructor, the default constructor will not automatically occur here. will need to have something like this 
```
public Card() {}
```

Reply:
at least for my code, an empty constructor is needed since i cannot get all field of the result card in the same method. So being able to make an empty instance would help.

Reply:
My understanding of the current structure of the code is that the Card class is just a container around these 4 fields, which should have no reason to change once they've been set. So I strongly recommend that this class be made immutable e.g. the fields are set in the constructor, which means there'd be no need for a default constructor or any setters.